### PR TITLE
Add example code for using ResultSet Scan

### PR DESCRIPTION
**Pull request by haoxins**
deps https://github.com/vesoft-inc/nebula-go/pull/305

## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

### CODE REVIEW

**Comment by haoxins** [comment]
cc @Nicole00

**Review by wey-gu — DISMISSED** [review]
Thanks!

File: result_set.go [hunk 1]
```diff
@@ -336,31 +336,35 @@ func (res ResultSet) Scan(v interface{}) error {
 
 // Scan scans the rows into the given value.
 func (res ResultSet) scanRow(row *nebula.Row, colNames []string, t reflect.Type) (reflect.Value, error) {
-	rowValues := row.GetValues()
+	rowVals := row.GetValues()
 
 	val := reflect.New(t).Elem()
```
Comment (wey-gu) [inline result_set.go:342]: loved this!

File: result_set.go [hunk 2]
```diff
@@ -357,8 +357,12 @@ func (res ResultSet) scanRow(row *nebula.Row, colNames []string, t reflect.Type)
 		rowVal := rowVals[cIdx]
 
 		switch f.Type.Kind() {
+		case reflect.Int:
+			val.Field(fIdx).SetInt(rowVal.GetIVal())
 		case reflect.Int64:
 			val.Field(fIdx).SetInt(rowVal.GetIVal())
+		case reflect.Float64:
+			val.Field(fIdx).SetFloat(rowVal.GetFVal())
```
Comment (Nicole00) [inline result_set.go:365]: don't we need to process bool/float32/int8/int16/date/datetime/time/duration/?

Comment (haoxins) [reply result_set.go:365]: we need, but I will open another PR to do this.
Also we need enhance/refactor the tests.
this is just make the example works.

**Comment by haoxins** [comment]
BTW, the examples are also complex for the starters.
We can add the most simple usage in the Readme.
Most use cases don't need to construct the space and schema,
we can give an example for the query only (query an already exists space with the defined schema).

**Comment by wey-gu** [comment]
> BTW, the examples are also complex for the starters. We can add the most simple usage in the Readme. Most use cases don't need to construct the space and schema, we can give an example for the query only (query an already exists space with the defined schema).

Thanks so much @haoxins , you are making NebulaGraph/ NebulaGraph community a lot better 🫡.

**Review by wey-gu — APPROVED** [review]

**Review by Nicole00 — APPROVED** [review]

**Pull request merged**